frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -284,6 +290,47 @@ const TableContainer = <T,>({
     onPaginationChange,
   ]);
 
+  const renderFilterActionButton = () => {
+    // always !!actionButton ehre, this is for type checker
+    if (actionButton) {
+      if (actionButton.gitOpsModeCompatible) {
+        return (
+          <GitOpsModeTooltipWrapper
+            tipOffset={8}
+            renderChildren={(disableChildren) => (
+              <Button
+                disabled={disableActionButton || disableChildren}
+                onClick={actionButton.onActionButtonClick}
+                variant={actionButton.variant || "brand"}
+                className={`${baseClass}__table-action-button`}
+              >
+                <>
+                  {actionButton.buttonText}
+                  {actionButton.iconSvg && <Icon name={actionButton.iconSvg} />}
+                </>
+              </Button>
```

Comment:
I'd just set this to a var and if `if (actionButton.gitOpsModeCompatible)` do `var = <GitOpsTooltipWrapper>{var}</GitOpsTooltipWrapper>` so we don't repeat the markup

Reply:
Sounds good

Reply:
It's a little trickier than that since `GitOpsModeTooltipWrapper`'s `disabled` prop needs the
additional `|| disableChildren` condition

Reply:
hm right -- how about always wrapping it in `<GitOpsModeTooltipWrapper>` but adding `actionButton.gitOpsModeCompatible` to the boolean in `disabled={...}`?

Reply:
It would still render the tooltip then

Reply:
(Logic for that is based on `GitOpsModeTooltipWrapper`'s internal reading of config context, can't be controlled by the `renderChildren` function)

Reply:
Gotcha.  Maybe an enhancement for a later date: an optional property to `<GitOpsModeTooltipWrapper>` to indicate whether it should make the check.  Also another point in favor of a future generic version where you could tell it which checks to make 🤐 .

Reply:
Absolutely....it's starting to just look like adding some funcitonality to good 'ol `TooltipWrapper` 🙂